src/widgets/Tabs/VaccineSelect.js

```diff
@@ -106,34 +113,37 @@ const VaccineSelectComponent = ({
 
   return (
     <FlexView style={pageTopViewContainer}>
-      <AfterInteractions placeholder={null}>
-        <FlexRow style={{ marginBottom: 7 }} justifyContent="flex-end">
-          <VaccinePrescriptionInfo />
-          <FormDropdown
-            options={UIDatabase.objects('MedicineAdministrator')}
-            optionKey="displayString"
-            label={vaccineStrings.vaccinator}
-            onValueChange={onSelectVaccinator}
-            value={vaccinator}
+      <FlexRow flex={1} justifyContent="flex-end">
+        <VaccinePrescriptionInfo />
+        <FormDropdown
+          options={UIDatabase.objects('MedicineAdministrator')}
+          optionKey="displayString"
+          label={vaccineStrings.vaccinator}
+          onValueChange={onSelectVaccinator}
+          value={vaccinator}
```

Comment:
this was getting big and I wanted to PR before digging into this drop down

Reply:
thanks